### ISSUE TITLE
Finding tweet from ID 

### ISSUE THREAD

**Issue by AbdullaRifai** [Jefferson-Henrique/GetOldTweets-python#268]
Does anyone know how I can get a tweet object from the ID only?

**Comment by rishi-raj-jain** [comment]
Yeah, I found a way around using selenium. Have a look at https://github.com/digitalPlayer1125/twitterUsernamefromUserID

**Comment by AbdullaRifai** [comment]
@digitalPlayer1125 thanks man! Been digging for a while for a way.
